GameOfRunes/GameOfRunes/Custom Nodes/ButtonNode/TapResponder.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  TapResponder.swift
+//  GameOfRunes
+//
+//  Created by Brian Yen on 11/3/20.
+//  Copyright © 2020 TeamHoWan. All rights reserved.
+//
+
+import SpriteKit
+
+/**
+ Conforming types can respond to tap events..
+ This should be implemented by the `SKScene` that the tapped node
+ is added to.
+ */
+protocol TapResponder: AnyObject {
```

Comment:
Hey, I think that `AnyObject` should be changed to `class`. See https://stackoverflow.com/questions/30176814/whats-the-difference-between-a-protocol-extended-from-anyobject-and-a-class-onl/32895975#32895975

Just a minor thing.

Reply:
o.O huh i thought most of the responses recommend `AnyObject` for protocol?

Reply:
eh psps i blur, i read it as AnyObject gonna be deprecated